RestartManager.js

```diff
@@ -0,0 +1,33 @@
+let NativeCodePush = require("react-native").NativeModules.CodePush;
+
+const RestartManager = (() => {
+    let _allowed = true;
+
+    function restartApp(onlyIfUpdateIsPending = false) {
+        if (_allowed) {
+            NativeCodePush.restartApp(onlyIfUpdateIsPending);
+        }
+    }
+
+    function allow() {
```

Comment:
Since I am now exposing the onlyIfUpdateIsPending a simple flag no longer suffices. Is there really a case where onlyIfUpdateIsPending is set to false? installUpdate always calls savePendingUpdate regardless of InstallMode. The only use of restartApp(false) I could find was in local package's install(). But that happens after installUpdate.